.github/workflows/conflict_labeler.yml

```diff
@@ -0,0 +1,24 @@
+name: Merge Conflict Labeler
+
+on:
+  push:
+    branches:
+      - develop
+  pull_request_target:
+  issue_comment:
```

Comment:
Is this necessary?

Reply:
the name is issue_comment but it work also for PR see screenshot 

![chrome_IgQXjJIRmF](https://github.com/Sonarr/Sonarr/assets/68083474/30418423-669c-4700-b616-06c0539c39dd)